radio/src/gui/colorlcd/lcd.cpp

```diff
@@ -69,11 +69,16 @@ static lv_disp_drv_t* refr_disp = nullptr;
 static void flushLcd(lv_disp_drv_t* disp_drv, const lv_area_t* area,
                      lv_color_t* color_p)
 {
-#if !defined(LCD_VERTICAL_INVERT) && !defined(BOOT)
-  // we're only interested in the last flush in direct mode
-  if (!lv_disp_flush_is_last(disp_drv)) {
-    lv_disp_flush_ready(disp_drv);
-    return;
+#if (!defined(LCD_VERTICAL_INVERT) || defined(RADIO_F16)) && !defined(BOOT) 
+#if defined(RADIO_F16)
+  if (hardwareOptions.pcbrev > 0)
```

Comment:
@richardclli what is behind this? Did they change the screen type from hardware revision to another?

Reply:
I think they rotated the screen between prototype and production version

Reply:
Don't know if it was between prototype or production, but Richard did say the screen was going to be rotated.